### DISCUSSION TITLE
Suggestion: Entity Hider Hide Dead NPC: Select NPC

### DISCUSSION THREAD

**Discussion by ghost** [runelite/runelite#15445] [Ideas]
Suggestion to add an option within entity hider to select which npcs to apply "Hide Dead NPCs" to (similar to NPC aggression timer options).

The only NPC I myself would like to hide death animations for are the Nylocas in Theatre. Additionally, within Inferno, if Hide Dead NPCs is turned on, it can cause issues with rangers as they can sometimes fire their attacks at the same time as the death animation starts.

**Comment by RsFoe** [comment]
yeah it would be awsome to be able to have the option to customize it. The MES option to hide attack options is much more pleasant for average gameplay but there certain situations where being able to choose between entity hider/MES hiding would be very helpful such as the inferno/fight caves and it sounds like it would be super useful in ToA. Nylocas would also be nice but if youre in a party its not a problem because of nylo death indicators, but there are also times when youre with random people and it would be nice to add nylocas to that list/and maiden nylocas to it.
